resources/assets/scripts/app/app.coffee

```diff
@@ -49,7 +51,7 @@ require '../shared/pfm-angular-sanitize'
 require '../shared/init.coffee'
 
 
-ponyfm = angular.module 'ponyfm', ['ui.bootstrap', 'ui.router', 'ui.date', 'ui.sortable', 'angularytics', 'ngSanitize', 'hc.marked']
+ponyfm = angular.module 'ponyfm', ['ui.bootstrap', 'ui.router', 'ui.date', 'ui.sortable', 'angularytics', 'ngSanitize', 'hc.marked', 'chart.js']
```

Comment:
The `chart.js` dependency should be injected into the controller that needs it instead of the top-level Angular module. This goes for its "require" statements, too.

This is a case of technical debt, where the existing way of "organizing" JS modules (throwing them all into `app.coffee`) is something of a hack job. Webpack was integrated into the asset pipeline to give us a way to use "proper" npm-style modules instead.